operate/common/src/main/java/io/camunda/operate/connect/ElasticsearchConnector.java

```diff
@@ -157,13 +165,13 @@ public void tearDown() {
     if (elasticsearchClient != null) {
       try {
         elasticsearchClient._transport().close();
-      } catch (IOException e) {
+      } catch (final IOException e) {
         throw new RuntimeException(e);
```

Comment:
:wrench: I would recommend re-throwing it as an `UncheckedIOException`, so the type still carries meaning.

Reply:
Changed here and in related OS connector code.